nameko_eventlog_dispatcher/eventlog_dispatcher.py

```diff
@@ -0,0 +1,124 @@
+import logging
+from copy import deepcopy
+from datetime import datetime, timezone
+
+from kombu.common import maybe_declare
+from nameko.amqp import get_connection, verify_amqp_uri
+from nameko.events import EventDispatcher
+from nameko.rpc import Rpc
+from nameko.standalone.events import event_dispatcher, get_event_exchange
+from nameko.web.handlers import HttpRequestHandler
+
+
+log = logging.getLogger(__name__)
+
+
+class EventLogDispatcher(EventDispatcher):
+
+    """Dispatcher of event logs reusing the Nameko `EventDispatcher`.
```

Comment:
It's inheriting from `EventDispatcher`, not reusing it. I'd leave this bit out, just `Dispatcher of event logs.` should be fine.